gis_exercise_2

```diff
@@ -66,7 +66,7 @@ geo.loc[0, 'geometry'] = poly
 # # P5. Save the GeoDataFrame into a new Shapefile called 'polygon.shp'
 # # --------------------------------------------------------------------
 # # Note: you do not need to define the coordinate reference system at this time
-outfp = r"C:\\Users\\stefa\\Documents\\Fortbildung privat\\Python\\GIS\\Data\\Exercise_2_Problem_1.shp"
+outfp = r"./outputs/results.Exercise_2_Problem_1.shp"
```

Comment:
yea, I have to understand how to adequately share my code... so here, you created a new directory "outputs"? Can I physically see (on github) where this directory is and what files are stored in it?

Reply:
Yes!, but you have to be in the right branch to see it:

<img width="586" alt="SGJJSR_Python_challenges_at_SGJJSR-exercise_2_py_problems" src="https://user-images.githubusercontent.com/37868/81504266-a0703e80-92e8-11ea-935a-2720be30e161.png">